charts/aws-az/aws-az-chaos/engine.yaml

```
@@ -0,0 +1,29 @@
+apiVersion: litmuschaos.io/v1alpha1
+kind: ChaosEngine
+metadata:
+  name: aws-az-chaos
+spec:
+  # It can be active/stop
+  engineState: 'active'
+  #ex. values: ns1:name=percona,ns2:run=nginx
+  auxiliaryAppInfo: ''
```

There is no check for aux app in the experiment, so we can remove this
```suggestion
  auxiliaryAppInfo: ''
```